regression/cbmc-primitives/forall_6231_2/test.c

```diff
@@ -0,0 +1,17 @@
+#include <assert.h>
+#include <stdlib.h>
+
+int main() {
+  char *a = malloc(128);
+
+  assert(
+    __CPROVER_forall {
+      int i ; (0 <= i && i < 1) ==> *(a+i) == *(a+i)
```

Comment:
Likewise; not just one instantiation please.

Reply:
Hi @martin-cs, I've left these as they stand, and added a separate test that does multiple instantiations (by essence of it having a bigger upper bound).

Does that sound like an adequate solution to you?

Reply:
It is OK but I think it might make these tests redundant.